test/strings/basic.jl

```diff
@@ -1472,3 +1472,6 @@ end
     u = b"hello"
     @test eltype(u) === UInt8
 end
+
+@test_throws MethodError codeunit("Julia", Int32(2))
```

Comment:
`codeunit("Julia", Int32(2))` works just fine for me, what did you intend this to test?